instance/models/load_balancer.py

```diff
@@ -147,7 +148,10 @@ def get_configuration(self):
         backend_map = []
         backend_conf = []
         for instance in self.get_instances():
-            map_entries, conf_entries = instance.get_load_balancer_configuration()
+            triggered_by_instance = instance.ref.pk == triggering_id
+            map_entries, conf_entries = instance.get_load_balancer_configuration(
+                triggered_by_instance
+            )
```

Comment:
@smarnach Would we be needing this complexity if the instances were responsible for calling the load balancer to add/update/remove their configuration?